pontoon/sync/changeset.py

```diff
@@ -195,6 +195,10 @@ def update_entity_translations_from_vcs(
             db_translations = db_entity.translation_set.filter(
                 locale__code=locale_code,
             )
+
+        if old_translations is None:
+            old_translations = db_translations.filter(approved_date__lte=self.now)
+
```

Comment:
Just moving this block up, closer to `db_translations`. Doesn't have any impact.